modules/platforms/dotnet/Apache.Ignite/IgniteServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,167 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Apache.Ignite;
+
+using System;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+
+/// <summary>
+/// Extension method for setting up Apache Ignite services in an <see cref="Microsoft.Extensions.DependencyInjection.IServiceCollection" />.
+/// </summary>
+public static class IgniteServiceCollectionExtensions
+{
+    /// <summary>
+    ///  Registers an <see cref="IgniteClientGroup" /> and an <see cref="IIgnite" />.
```

Comment:
I don't think we can register `IIgnite` in the DI container, because it requires async initialization through `IgniteClientGroup.GetIgniteAsync()`.

Reply:
Yes, that's an artifact which will be removed

Reply:
Fixed